src/Downloader.ts

```diff
@@ -447,6 +447,9 @@ class Downloader {
     logger.info(`Getting article [${articleId}] from ${articleApiUrl}`);
 
     const json = await this.getJSON<any>(articleApiUrl);
+    if (json.error) {
```

Comment:
Why not putting this in getJSON()?

Reply:
getJSON is used for getting other things as well so if we put it there and we want to skip it that won't happen

Reply:
OK, but if I understand well, the `json` variable, does not have to be JSON? What about if this is `undefined`? Which part of the code set `json.error` (is that a javascript internal? Or just a value coming from the Mediawiki API?